consensus/bitcoin-ng.md

```diff
@@ -1,22 +1,13 @@
 Bitcoin-NG for Aeternity
 ==========
 
-Bitcoin-NG is a new consensus algorithm [1]. It depends on Proof-of-Work and uses chain of blocks with the most work done to resolve conflicts. Aeternity is PoW blockchain. Using Bitcoin-NG is very suitable for reducing on-chain confirmation latency. Bitcoin-NG is based on temporary leader that is elected randomly using Proof-of-Work.
-
-
-How to understand Bitcoin-NG?
-===
-
-To understand NG really quickly, we need to re-cap how we can think about classic Nakamoto consensus.
-
-Nakamoto consensus is based on crypto-puzzle that chooses a leader (miner) who confirms all the events included in the past block. New events are aggregated by all miners until one of them confirms them.
-
-Bitcoin-NG consensus is based on crypto-puzzle that chooses a leader (miner) who confirms all the events that will happen until the next miner solves the crypto-puzzle. The next miner takes over leadership.
+Bitcoin-NG is an evolution of Bitcoin protocol [1]. It uses the chain of blocks with the most work done to establish consensus. Consensus is guarded with Proof of Work. Using Bitcoin-NG improves latency of achieving on-chain consensus. Bitcoin-NG is based on temporary leader that is elected randomly using Proof-of-Work.
 
+There are two types of blocks in BitcoinNG. Key Blocks require Proof of Work solution. Key Blocks don't carry any transaction. The miners who solves a Key Block becomes the leader. Leaders generate another type of blocks called Micro Blocks that contain transactions. The latency of Micro Blocks is limited by network propagation delay, however we set minimum broadcast time to reduce malicious behaviour.
```

Comment:
Remove this paragraph in favour of the previous one. The time limit should be mentioned in the in depth section